### PR TITLE
I 100% tested this and this 100% works please stop dming me Tanzanite

### PR DESCRIPTION
**Pull request by kr45732**
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite
I 100% tested this and this 100% works please stop dming me Tanzanite

### CODE REVIEW

**Comment by IRONM00N** [comment]
this is such a shitty way to do it

**Comment by kr45732** [comment]
please fix then thanks

**Comment by kr45732** [comment]
very good pr, merge!

File: src/lib/common/HighlightManager.ts [hunk 1]
```diff
@@ -211,6 +211,11 @@ export class HighlightManager {
 	public async notify(message: BushMessage, user: Snowflake, hl: HighlightWord): Promise<boolean> {
 		assert(message.inGuild());
 
+		const userObject = await message?.guild?.members?.cache?.get(user);
```
Comment (romangraef) [inline src/lib/common/HighlightManager.ts:214]: why would you block highlight while online, people arent always monitoring every guilds general. this should at least be a toggleable option

**Review by romangraef — CHANGES_REQUESTED** [review]
this doesnt feel li

**Pull request closed without merging**